ghost/values.yaml

```diff
@@ -0,0 +1,294 @@
+  tag: 3.8.0-debian-10-r0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## String to partially override ghost.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override ghost.fullname template
+##
+# fullnameOverride:
+
+## Init containers parameters:
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+volumePermissions:
+  image:
+    registry: docker.io
+    repository: bitnami/minideb
+    tag: buster
+    pullPolicy: Always
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+## Ghost protocol, host, port and path to create application URLs
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+##
+ghostProtocol: http
+# ghostHost:
+# ghostPort:
+ghostPath: /
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+##
+ghostUsername: user@example.com
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+##
+# ghostPassword:
+
+## Admin email
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+##
+ghostEmail: user@example.com
+
+## Ghost Blog name
+## ref: https://github.com/bitnami/bitnami-docker-ghost#environment-variables
+##
+ghostBlogTitle: User's Blog
+
+## Set to `yes` to allow the container to be started with blank passwords
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+allowEmptyPassword: "yes"
+
+## SMTP mail delivery configuration
+## ref: https://github.com/bitnami/bitnami-docker-redmine/#smtp-configuration
+##
+# smtpHost:
+# smtpPort:
+# smtpUser:
+# smtpPassword:
+# smtpFromAddress
+# smtpService:
+
+## Configure extra options for liveness and readiness probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
+##
+livenessProbe:
+  enabled: true
+  initialDelaySeconds: 120
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+readinessProbe:
+  enabled: true
+  initialDelaySeconds: 30
+  periodSeconds: 5
+  timeoutSeconds: 3
+  failureThreshold: 6
+  successThreshold: 1
+
+##
+## External database configuration
+##
+externalDatabase:
```

Comment:
I think we can remove `externalDatabase`

Reply:
I just tested without externalDatabase, When the user uninstalls helm application then there is a database left that can be connected later on with the external database. But without it the chart gives an error that it is not able to connect MariaDB (Which inturn is then become external database)

Reply:
Makes sense, good point!